src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -847,7 +855,7 @@ protected void onPause() {
             mDownloadFinishReceiver = null;
         }
         
-        
+        PinCheck.setUnlockTimestamp();
```

Comment:
This is correctly paired with onResume(), but should move to onStop() when the check is moved to onStart().